moped-editor/package.json

```diff
@@ -100,7 +100,8 @@
     "npm": ">=6.0.0"
   },
   "devDependencies": {
+    "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
```

Comment:
this fixes this problem:
https://stackoverflow.com/questions/76435306/babel-preset-react-app-is-importing-the-babel-plugin-proposal-private-propert